static/src/javascripts/projects/common/modules/commercial/badges.js

```diff
@@ -69,8 +69,13 @@ define([
 
             return new Promise(function (resolve) {
                 idleFastdom.write(function () {
-                    $('.js-container__header', container)
-                        .after($adSlot);
+                    var placeholder = $('.js-badge-placeholder', container);
+
+                    if (placeholder[0]) {
+                        placeholder.replaceWith($adSlot[0]);
```

Comment:
@regiskuckaertz I think that is less weird than only replacing the first candidate.